arm-graphrbac/1.6/swagger/graphrbac.json

```diff
@@ -111,14 +111,14 @@
           "Application"
         ],
         "operationId": "Applications_List",
-        "description": "Lists applications by filter parameters. Reference: http://msdn.microsoft.com/en-us/library/azure/hh974476.aspx",
+        "description": "Lists applications by filter parameters. For more information, see http://msdn.microsoft.com/library/azure/hh974476.aspx.",
```

Comment:
@erickson-doug There are MSDN links like this in most of the operation descriptions. Keep or delete?